conf.yaml

```diff
@@ -33,6 +33,7 @@ repos:
     kibana-cn:            https://github.com/elasticsearch-cn/kibana.git
     logstash:             https://github.com/elastic/logstash.git
     logstash-docs:        https://github.com/elastic/logstash-docs.git
+    security-docs:        https://github.com/elastic/security-docs.git
```

Comment:
This repo doesn't exist?
https://github.com/elastic/security-docs

Reply:
@bmorelli25: Might be due to the repo currently being private. Let me switch to the public and see if that fixes the preview.